anesthetic/samples.py

```diff
@@ -355,6 +355,8 @@ def plot_2d(self, axes=None, *args, **kwargs):
                         else:
                             ax.plot([], [])
 
+        axes._set_logticks()
```

Comment:
I tried setting the log ticks in the same place where we generally deal with tick locations, but `plot_2d` overwrites these locations, so this call at the end of `plot_2d` is needed...